server/views/pages/reportDetailMacro.njk

```diff
@@ -200,15 +200,17 @@
         print: print
       })
     }}
+    
+    {# painInducingTechniques was not a question in earlier iterations of the app hence the 'if' to accommodate backwards compatibility #}
     {% if data.useOfForceDetails.painInducingTechniques != undefined  %}
-    {{
-      reportDetailsMacros.tableRow({
-        label: 'Were pain inducing techniques used?',
-        'data-qa': 'painInducingTechniques',
-        dataValue: data.useOfForceDetails.painInducingTechniques | toYesNo,
-        print: print
-      })
-    }}
+      {{
+        reportDetailsMacros.tableRow({
+          label: 'Were pain inducing techniques used?',
+          'data-qa': 'painInducingTechniques',
+          dataValue: data.useOfForceDetails.painInducingTechniques | capitalize,
```

Comment:
Hmm. This might explain the lower case 'yes' and 'no' in reportSummary.js - but that file uses 'Yes' and 'No' everywhere else. I think we should either use 'Yes' and 'No' or 'yes' and 'no', not both. In fact those values should be extracted to constants.

Reply:
Have changed them to Yes and No